go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/ory/dockertest/v3
+
+go 1.13
+
```

Comment:
This is missing the rewrite though, right?

Reply:
That's right, I thought I'd get the modules port done first and then see if rewrites were necessary at the dockertest level: I'll probably raise a separate PR for that once this is in. I may end up doing the code change to work with the newer version instead, if that's straightforward.

Reply:
SGTM :)